bootstrap/main.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import pathlib
+import shutil
+import sys
+import time
+from typing import Any
+
+import docker
+
+client = docker.from_env()
+
+default_file_path = pathlib.Path("/startup.json.default")
+config_file_path = pathlib.Path("/config/startup.json")
+
+CORE_CONTAINER_NAME = "companion_core"
+
+
+def ensure_dir(file_path: str) -> None:
+    """Makes sure that the file path exists
+
+    Args:
+        file_path (str): path to check/make
+    """
+    directory = os.path.dirname(file_path)
```

Comment:
since you are using pathlib, I would recommend to not use `os.path`.
E.g: `os.path.dirname` is `path.parent`, `os.path.exists` is `path.exists()` and `os.path.makedirs` is `path.mkdir`.
For reference: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module